bot/modules/roll/commands/roll.js

```diff
@@ -94,7 +117,8 @@ module.exports = class extends Command
     let total = eval(operation);
     total = Math.round(total);
 
-    context.chat(`Breakdown: ${breakdown}\nTotal: ${total}`);
+    label = hasText.test(label) ? label : 'Result';
+    context.chat(`Breakdown: ${breakdown}\n**${label}**: ${total}`);
```

Comment:
Try mo gamitin `context.send(tittle, description)`.
Ginagawa nito nagsesend ng embed parang ganito...
![image](https://user-images.githubusercontent.com/18418806/85920603-bab9a800-b8a7-11ea-9806-32f4e318ef91.png)

'Yong nakabold text sa pic ("Wheel of TWICE"), 'yon 'yong first param `title`...
'Yong other text ("You chose Jihyo..."), 'yon 'yong second param `description`. Pwede Discord markdown dito.

Suggestion lang naman 'to.

Reply:
Ayt ayt

Reply:
Resolved this as `context.chat()` to keep it quotable.